modules/nw-ptp-dual-wpc-hardware-config-reference.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-ptp-dual-wpc-hardware-config-reference_{context}"]
+= Intel Westport Channel E810 dual NIC hardware configuration reference
```

Comment:
Minor noticing, but I note that another topic is titled "Configuring linuxptp services as a grandmaster clock for dual E810 Westport Channel NICs" and I wonder if the title here should swap around the E810 and dual. #notmyprodarea so I defer to you. Just a minor thing I noticed.